intelmq/bots/experts/fqdn2ip/expert.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+import sys
+import socket
+
+import intelmq.lib.harmonization
+from intelmq.lib.bot import Bot
+
+
+class fqdn2ipExpertBot(Bot):
+
+    def process(self):
+        event = self.receive_message()
+
+        for key in ["source.", "destination."]:
+            key_fqdn = key + "fqdn"
+            key_ip = key + "ip"
+            if not event.contains(key_fqdn):
+                continue
+            if event.contains(key_ip):
+                continue
+            ip = socket.gethostbyname(event.get(key_fqdn))
+            if intelmq.lib.harmonization.IPAddress.is_valid(ip, sanitize=True):
+                event.add(key_ip, ip, sanitize=True, force=True)
```

Comment:
`sanitize` is True by default.
Why do you force? If the IP exists, this code is not executed anyway.